indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -4,21 +4,55 @@ <h6>
     {% if work.stub %}
       <span class="badge badge-info">stub</span>
     {% endif %}
+    <br/><span class="text-muted mt-1">{{ work.frbr_uri }}</span>
```

Comment:
```suggestion
    <br><span class="text-muted mt-1">{{ work.frbr_uri }}</span>
```

HTML5 doesn't actually support self-closing tags (that's an XML thing).